app/assets/stylesheets/entries.css

```diff
@@ -0,0 +1,40 @@
+.content-container {
+    display: flex;
```

Comment:
Indentation should be 2 spaces, you can try using the default formatter of your editor or any good extension.

Reply:
okey thank you i changed the default indentation to 2 Spaces, hope this works now

Reply:
Indentations are still not fixed for the css files ..